docs/CODE_OF_CONDUCT-bn.md

```diff
@@ -1,25 +1,26 @@
-অংশগ্রহণকারীদের দ্বারা অগ্রহণযোগ্য আচরণের উদাহরণগুলির মধ্যে রয়েছে:
+অভিজ্ঞতার স্তর, লিঙ্গ, লিঙ্গ পরিচয় এবং অভিব্যক্তি, যৌন অভিযোজন, অক্ষমতা, ব্যক্তিগত চেহারা, শরীরের আকার, জাতীয়তা, জাতি, বয়স, ধর্ম, নির্বিশেষে আমরা এই প্রকল্পে অংশগ্রহণকে প্রত্যেকের জন্য একটি হয়রানি-মুক্ত অভিজ্ঞতা করতে প্রতিশ্রুতিবদ্ধ।
 
-* যৌনতামূলক ভাষা বা চিত্রের ব্যবহার
-* ব্যক্তিগত আক্রমণ
-* ট্রোলিং বা অপমানজনক/অপমানজনক মন্তব্য
-* সরকারি বা ব্যক্তিগত হয়রানি
-* স্পষ্ট অনুমতি ছাড়াই অন্যের ব্যক্তিগত তথ্য, যেমন শারীরিক বা ইলেকট্রনিক ঠিকানা প্রকাশ করা
-* অন্যান্য অনৈতিক বা পেশাগত আচরণ ।
+**অংশগ্রহণকারীদের দ্বারা অগ্রহণযোগ্য আচরণের উদাহরণগুলির মধ্যে রয়েছে:**
 
+- যৌনতামূলক ভাষা বা চিত্রের ব্যবহার
+- ব্যক্তিগত আক্রমণ
+- ট্রোলিং বা অপমানজনক মন্তব্য
+- জনসম্মুখে  বা ব্যক্তিগত হয়রানি
+- স্পষ্ট অনুমতি ছাড়াই অন্যের ব্যক্তিগত তথ্য, যেমন শারীরিক বা ইন্টারনেট ঠিকানা প্রকাশ করা
+- অন্যান্য অনৈতিক বা পেশাগত আচরণ
 
-প্রজেক্ট রক্ষণাবেক্ষণকারীদের এই আচরণবিধির সাথে সংযুক্ত নয় এমন মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা, সমস্যা এবং অন্যান্য অবদানগুলিকে অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব রয়েছে, বা অন্যান্য আচরণের জন্য অস্থায়ী বা স্থায়ীভাবে কোনো অবদানকারীকে নিষিদ্ধ করার অধিকার রয়েছে। তারা অনুপযুক্ত, হুমকি, আপত্তিকর, বা ক্ষতিকারক বলে মনে করে।
+প্রজেক্ট রক্ষণাবেক্ষণকারীদের এই আচরণবিধির সাথে সংযুক্ত নয় এমন মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা, সমস্যা এবং অন্যান্য অবদানগুলিকে অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব রয়েছে, বা অন্যান্য আচরণের জন্য অস্থায়ী বা স্থায়ীভাবে কোনো অবদানকারীকে নিষিদ্ধ করার অধিকার রয়েছে যা তারা অনুপযুক্ত, হুমকি, আপত্তিকর, বা ক্ষতিকারক বলে মনে করে।
 
 এই আচরণবিধি গ্রহণ করার মাধ্যমে, প্রকল্পের রক্ষণাবেক্ষণকারীরা এই প্রকল্প পরিচালনার প্রতিটি ক্ষেত্রে এই নীতিগুলিকে ন্যায্যভাবে এবং ধারাবাহিকভাবে প্রয়োগ করার জন্য নিজেদের প্রতিশ্রুতিবদ্ধ করে। প্রজেক্ট রক্ষণাবেক্ষণকারী যারা আচরণবিধি অনুসরণ করে না বা প্রয়োগ করে না তাদের প্রকল্প দল থেকে স্থায়ীভাবে সরিয়ে দেওয়া হতে পারে।
 
 এই আচরণবিধি প্রজেক্ট স্পেস এবং পাবলিক স্পেস উভয় ক্ষেত্রেই প্রযোজ্য হয় যখন একজন ব্যক্তি প্রকল্প বা তার সম্প্রদায়ের প্রতিনিধিত্ব করেন।
 
-gmail.com-এ victorfelder-এ একজন প্রজেক্ট রক্ষণাবেক্ষণকারীর সাথে যোগাযোগ করে আপত্তিজনক, হয়রানি, বা অন্যথায় অগ্রহণযোগ্য আচরণের ঘটনাগুলি রিপোর্ট করা যেতে পারে। সমস্ত অভিযোগ পর্যালোচনা করা হবে এবং তদন্ত করা হবে এবং পরিস্থিতির জন্য প্রয়োজনীয় এবং উপযুক্ত বলে মনে করা হবে। রক্ষণাবেক্ষণকারীরা একটি ঘটনার প্রতিবেদকের বিষয়ে গোপনীয়তা বজায় রাখতে বাধ্য।
 
-এই কোড অফ কন্ডাক্ট কন্ট্রিবিউটর কভেন্যান্ট, সংস্করণ 1.3.0 থেকে অভিযোজিত হয়েছে, এখানে উপলব্ধ।
- https://contributor-covenant.org/version/1/3/0/
-Translations
+কেও যদি আপত্তিজনক, হয়রানি বা অগ্রহণযোগ্য আচরণের শিকার হয়, তাহলে যে কোন একজন প্রজেক্ট রক্ষণাবেক্ষণকারীর সাথে `victorfelder[@]gmail.com` যোগাযোগ করার অনুরোধ করা হল। যোগাযোগ কারীর তথ্য অবশ্যই গোপন রাখা হবে। 
```

Comment:
remove যে কোন একজন, because the original document mentions the particular owner. Also, that is grammatically wrong.

Reply:
ok

Reply:
@souravdasslg Give it a review again please.